sources/PTYSession.m

```diff
@@ -205,6 +205,8 @@ @implementation PTYSession {
     VT100Terminal *_terminal;
 
     NSString *_termVariable;
+    
+    NSString *_answerBackString;
```

Comment:
PTYSession doesn't need to keep a copy of `answerBackString`. It's ok for VT100Terminal to keep the only copy of it. So you can get rid of this ivar and get rid of `setAnswerbackString:` and just set it directly in the VT100Terminal.